b-real-time-htb.js

```diff
@@ -233,24 +257,22 @@ function BRealTimeHtb(configs) {
 
         /* ---------- Proces adResponse and extract the bids into the bids array ------------*/
 
-        var bids = adResponse;
+        var bids = adResponse.tags;
 
         /* --------------------------------------------------------------------------------- */
-
         for (var i = 0; i < bids.length; i++) {
 
-            var curReturnParcel;
-
+            var curReturnParcel, bid = bids[i];
+            
             for (var j = unusedReturnParcels.length - 1; j >= 0; j--) {
-
+                var unusedReturnParcel = unusedReturnParcels[j];
```

Comment:
It looks like this variable is only used once, can you please remove this variable and use `unusedReturnParcels[j]` directly?